zipline/pipeline/loaders/blaze/utils.py

```diff
@@ -0,0 +1,61 @@
+from zipline.pipeline.common import SID_FIELD_NAME, TS_FIELD_NAME
+from zipline.pipeline.loaders.blaze.core import ffill_query_in_range
+from zipline.pipeline.loaders.utils import (
+    normalize_data_query_bounds,
+    normalize_timestamp_to_query_time,
+)
+
+
+def load_raw_data(assets, dates, data_query_time, data_query_tz, expr,
+                  odo_kwargs):
+    """
+    given an expression representing data to load, perform normalization and
+    forward-filling and return the data, materialized.
+
+    parameters
+    ----------
+    assets : pd.int64index
+        the assets to load data for.
+    dates : pd.datetimeindex
+        the simulation dates to load data for.
+    data_query_time : datetime.time
+        the time used as cutoff for new information.
+    data_query_tz : tzinfo
+        the timezone to normalize your dates to before comparing against
+        `time`.
+    expr : expr
+        the expression representing the data to load.
+    odo_kwargs : dict, optional
+        extra keyword arguments to pass to odo when executing the expression.
+
+    returns
+    -------
+    raw : pd.dataframe
+        the data symbolized by `expr` materialized in a dataframe.
```

Comment:
Could we say: 'The result of computing expr and materializing the result as a dataframe'.